### PR TITLE
Propogate URL scheme (Fix test_composition.py)

### PR DESCRIPTION
**Pull request by mellenburg**
use protocol consistent with dns

### CODE REVIEW

File: packages/dcos-integration-test/extra/conftest.py [hunk 1]
```diff
@@ -240,6 +240,7 @@ def __init__(self, dcos_uri, masters, public_masters, slaves, public_slaves,
 
         # URI must include scheme
         assert dcos_uri.startswith('http')
+        self.protocol = dcos_uri.split(':')[0]
```
Comment (cmaloney) [inline packages/dcos-integration-test/extra/conftest.py]: If we want to start parsing up the uri, we should do with: https://docs.python.org/3/library/urllib.parse.html

**Comment by mellenburg** [comment]
@cmaloney changed to use urllib.parse

**Comment by mellenburg** [comment]
@spahl @cmaloney this is ready for review/merge

**Comment by spahl** [comment]
:ship:

**Pull request merged**